test/functions/hilde/defaultOf/precedence.chpl

```diff
@@ -19,7 +19,7 @@ proc _defaultOf(type t) where t == R
   return result;
 }
 
-proc R.R()
+proc R.init()
```

Comment:
That seems reasonable to me.

We know that _defaultOf is still applied to generic classes and records (I think), and there is at least one more spot that is inserting a call to it for concrete classes and records when we don't want it to do so, but we seem to be moving away from it otherwise